plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/RocksDBKeyValueStorageFactory.java

```diff
@@ -145,7 +145,12 @@ private void init(final BesuConfiguration commonConfiguration) {
     try {
       databaseVersion = readDatabaseVersion(commonConfiguration);
     } catch (final IOException e) {
-      throw new StorageException("Failed to retrieve the RocksDB database meta version", e);
+      final String message =
+          e.getMessage() != null
+              ? "Failed to retrieve the RocksDB database meta version: " + e.getMessage()
+              : "Failed to retrieve the RocksDB database meta version";
```

Comment:
I think we could print the e.getMessage even if it is null, and remove the if statement and string duplication

Reply:
Done